components/layout/index.js

```diff
@@ -1,6 +1,7 @@
 import Head from 'next/head';
 import PropTypes from 'prop-types';
 import Footer from 'components/footer';
+import Navbar from 'components/navbar';
 const Layout = (props) => {
   const { children, title, description } = props;
```

Comment:
Since it receives only 3 props, we can destructure it while declaring the function
```javascript
const Layout = ({ title, description, children }) => {
```

Reply:
@sumitd94  ok will do it.

Reply:
@sumitd94 Please review it and suggest the changes if needed.